modules/widgets/top-posts.php

```diff
@@ -319,6 +319,13 @@ function widget( $args, $instance ) {
 			return;
 		}
 
+		$layout = apply_filters('jetpack_top_posts_widget_layout', '', $posts, $display );
+		if (!empty($layout)) {
```

Comment:
We try to use WordPress coding standards that require spaces before and after braces, before and after operators. This line would look like this: `if ( ! empty( $layout ) ) {`